test/qa/test_connect6.py

```diff
@@ -12,99 +12,81 @@
     login,
     network,
 )
+from test_connect import disconnect_base_test
 
 
-def setup_module(module):  # noqa: ARG001
+def setup_function(function):  # noqa: ARG001
     daemon.start()
     login.login_as("default")
+    logging.log()
+    print(sh.nordvpn.set.ipv6.on())
+
 
+def teardown_function(function):  # noqa: ARG001
+    logging.log(data=info.collect())
+    logging.log()
 
-def teardown_module(module):  # noqa: ARG001
+    print(sh.nordvpn.set.ipv6.off())
```

Comment:
same here with try/catch?

Reply:
I would like to disagree. If these fail, it could indicate a big issue with our app, and we'd miss it, just because we have try/catch block.

Furthermore, even if exception would be caught here, within next test we'd get an exception @ line 22, where we enable IPv6. So in that case, we'd also need to add try/catch block in `setup_function`, which means, that issues for example with IPv6 feature may be unnoticed for longer period of time :smile: